apps/client/pages/calendar.tsx

```diff
@@ -104,11 +104,13 @@ const Calendar = () => {
           </Button>
         </HStack>
       </VStack>
-      <TaskModal
-        isOpen={modalOpened}
-        task={clickedTask}
-        close={() => setModalOpened(false)}
-      />
+      {modalOpened && (
+        <TaskModal
+          isOpen={modalOpened}
+          task={clickedTask}
+          close={() => setModalOpened(false)}
+        />
+      )}
```

Comment:
why did You put modalOpened at the beginning?

Reply:
if this is because of the spinner that occured below eg. calendar view - it's fixed in my pr with google button - if it's because of anything else please clarify

Reply:
Yes, that's because you were not hiding the spinner